### ISSUE TITLE
CUDA Problem

### ISSUE THREAD

**Issue by halukgun** [fidler-lab/defgrid-release#2]
Hi, When I try to train the model on v100 16 GB GPU, I confront with CUDA problem. To solve this issue, I just change the --resolution as 256 512. I can run the model with this --resolution successfully. On other hand, the loss is 
![Screen Shot 2020-10-08 at 19 28 00](https://user-images.githubusercontent.com/37690108/96349644-7c93fe80-10b9-11eb-95ff-2155ae7a7c00.png)
As you can see, values are fluctuating. How can I solve this issue? Which GPUs are you used when you are doing training?

**Comment by SteveJunGao** [comment]
Hi!

I'm not sure I got the question clearly. If the values are fluctuating in the training set, then it makes sense, as different training images can have different variance&reconstruction error, since the RGB values are different.

Can you show the average variance on the whole validation set for each epoch? In my case, that value is converging after several epochs.  

As shown in README, I tested the code in TITAN X/Xp and GTX 1080Ti GPU

**Comment by 654493176** [comment]
Hi,I have the same question,I can run the model but the loss look like 
![image](https://user-images.githubusercontent.com/40239724/118650244-ccd16b80-b816-11eb-9163-f5e87b0fc25e.png)
I'm sure the input data is right.Have you solved your problem yet?would you prefer to give me some help?
Thank you!

**Comment by SteveJunGao** [comment]
Hi, is this the training curve or validation curve? typically it's normal to see this effect during training, but inference would be stable
